doc/api/util.md

```diff
@@ -1518,25 +1518,22 @@ const options = {
   'logfile': { type: 'string' },
   'no-logfile': { type: 'boolean' },
 };
-const { values, tokens } = parseArgs({ options, tokens: true });
+const { values, tokens = [] } = parseArgs({ options, tokens: true });
```

Comment:
This does not seem needed for this example. Always get an array back if ask for `tokens`?
```suggestion
const { values, tokens} = parseArgs({ options, tokens: true });
```

Reply:
Good question, I was trying the original code example in a plain javascript file with runtime error:

```
TypeError: Cannot read properties of undefined (reading 'filter')
```

I then take the snippets into typescript file with `@types/node@18` installed, expected the type checking would caught it before the runtime, sadly I got green light from **tsc** but runtime errors same as above.

Later found out that `tokens` support was only added since Node.js **v18.7** but I only had **v18.3** at the time, thus motived me to modified code to assign empty array as its default value just in case.

I double checked type definition afterwards, it turns out the typing could correctly infer between
`{ options, tokens: true }`, `{ options, tokens: false }` and even only `{ options }` !

I think it's good thing for user with correct tooling setup, but I'm not entirely sure the example code in docs should have such assumptions, thoughts?

Reply:
Thanks for back-story.

That is a scenario that someone else could fall into. But adding a default empty array turns a noisy failure into a silent failure without addressing the issue.

The documentation web page is titled with the node version it describes. I think there is an explicit expectation that the example code is for that version. The `parseArgs` feature is marked as experimental in v18 so subject to breaking change between v18 versions, and checking the history is needed to see what has changed.

Reply:
Agreed.